### PR TITLE
Update ReadMe 

### PR DESCRIPTION
**Pull request by skanwal**
Added instructions for running RNAsum via AWS-Batch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
...
-  	 * [WTS WGS and clinical data](#3-wts-wgs-and-clinical-data)
-  * [Output](#output)
+- [RNAsum](#rnasum)
+  - [Table of contents](#table-of-contents)
+  - [Installation](#installation)
+  - [Docker](#docker)
+  - [Workflow](#workflow)
+  - [Reference data](#reference-data)
+    - [External reference cohorts](#external-reference-cohorts)
+          - [Note](#note)
```
Comment (JMarzec) [inline README.md:16]: Do we need to list "notes"? I think these are just additional information that don't need to be listed in TOC

File: README.md [hunk 2]
```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
...
+    - [Internal reference cohort](#internal-reference-cohort)
+          - [Note](#note-1)
```
Comment (JMarzec) [inline README.md:18]: as above, do we need to list "notes"?

File: README.md [hunk 3]
```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
...
+  - [Input data](#input-data)
+    - [WTS](#wts)
+      - [bcbio-nextgen](#bcbio-nextgen)
+          - [Note](#note-2)
```
Comment (JMarzec) [inline README.md:22]: as above, do we need to list "notes"?

File: README.md [hunk 4]
```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
...
+      - [Dragen RNA](#dragen-rna)
+    - [WGS](#wgs)
+  - [Usage](#usage)
+    - [Arguments](#arguments)
+          - [Note](#note-3)
```
Comment (JMarzec) [inline README.md:27]: as above, do we need to list "notes"?

File: README.md [hunk 5]
```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
...
+    - [Run via AWS-Batch](#run-via-aws-batch)
+    - [Examples](#examples)
+          - [Note](#note-4)
```
Comment (JMarzec) [inline README.md:30]: as above, do we need to list "notes"?

File: README.md [hunk 6]
```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
...
-     * [Dragen RNA](#dragen-rna)
-  * [WGS](#wgs)
-* [Usage](#usage)
-  * [Arguments](#arguments)
-  * [Examples](#examples)
-  	 * [WTS data only](#1-wts-data-only)
-  	 * [WTS and WGS data](#2-wts-and-wgs-data)
-  	 * [WTS WGS and clinical data](#3-wts-wgs-and-clinical-data)
-  * [Output](#output)
+- [RNAsum](#rnasum)
```
Comment (JMarzec) [inline README.md:9]: I think we can skip that TOC header, it's all about RNAsum

File: README.md [hunk 7]
```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
...
+  - [Table of contents](#table-of-contents)
```
Comment (JMarzec) [inline README.md:10]: And I think that we don't need to mention TOC within TOC?

File: README.md [hunk 8]
```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
...
+        - [bcbio-nextgen](#bcbio-nextgen-1)
+        - [Dragen RNA](#dragen-rna-1)
+      - [2. WTS and WGS data](#2-wts-and-wgs-data)
+        - [bcbio-nextgen](#bcbio-nextgen-2)
+        - [Dragen RNA](#dragen-rna-2)
+      - [3. WTS WGS and clinical data](#3-wts-wgs-and-clinical-data)
+        - [bcbio-nextgen](#bcbio-nextgen-3)
+        - [Dragen RNA](#dragen-rna-3)
+    - [Output](#output)
+      - [Report](#report)
```
Comment (JMarzec) [inline README.md:41]: I'd skip that part in the TOC...

File: README.md [hunk 9]
```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
...
+      - [Results](#results)
```
Comment (JMarzec) [inline README.md:42]: I'd skip that part in the TOC...

**Review by JMarzec — COMMENTED** [review]
Thanks for updating info about running RNAsum using AWS-Batch! I'd only make some changes in the TOC

File: README.md (hunk 9 above)
Comment (skanwal) [reply README.md]: Why do you want to skip this section?

File: README.md (hunk 8 above)
Comment (skanwal) [reply README.md]: And why is so?

File: README.md (hunk 7 above)
Comment (skanwal) [reply README.md]: Fixed in https://github.com/umccr/RNAsum/pull/46/commits/16dfd3d474ef85becc66ddcd67da1b44dd38f0da

File: README.md (hunk 6 above)
Comment (skanwal) [reply README.md]: Updated in https://github.com/umccr/RNAsum/pull/46/commits/16dfd3d474ef85becc66ddcd67da1b44dd38f0da

File: README.md (hunk 5 above)
Comment (skanwal) [reply README.md]: Updated in https://github.com/umccr/RNAsum/pull/46/commits/16dfd3d474ef85becc66ddcd67da1b44dd38f0da

File: README.md (hunk 4 above)
Comment (skanwal) [reply README.md]: Updated in https://github.com/umccr/RNAsum/pull/46/commits/16dfd3d474ef85becc66ddcd67da1b44dd38f0da

File: README.md (hunk 3 above)
Comment (skanwal) [reply README.md]: Updated in https://github.com/umccr/RNAsum/pull/46/commits/16dfd3d474ef85becc66ddcd67da1b44dd38f0da

File: README.md (hunk 2 above)
Comment (skanwal) [reply README.md]: Updated in https://github.com/umccr/RNAsum/pull/46/commits/16dfd3d474ef85becc66ddcd67da1b44dd38f0da

File: README.md (hunk 1 above)
Comment (skanwal) [reply README.md]: updated in https://github.com/umccr/RNAsum/pull/46/commits/16dfd3d474ef85becc66ddcd67da1b44dd38f0da

**Comment by skanwal** [comment]
I am not manually generating table of contents. I use a markdown library in my code editor to generate it.
Just so you know, if you don't want to add anything to heading then either not append it with `#` or use `<!-- omit in toc -->`, otherwise it plays with the table of contents generator.

File: README.md (hunk 8 above)
Comment (JMarzec) [reply README.md]: The "Results" paragraph is really short, just one sentence. But I'm happy to keep it in the TOC if you feel that's relevant

File: README.md (hunk 9 above)
Comment (JMarzec) [reply README.md]: The "Results" paragraph is really short, just one sentence. But I'm happy to keep it in the TOC if you feel that's relevant

**Pull request merged**